strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/service/impl/XmlUserService.java

```diff
@@ -98,8 +61,8 @@ public void setUsers(final UsersDto newUsers)
     @Documented
     @Retention(RUNTIME)
     @Qualifier
-    public @interface StrongboxUserServiceQualifier
+    public @interface Xml
     {
-        String value() default "strongboxUserServiceQualifier";
+        String value() default "userService.xml";
```

Comment:
Would you mind clarifying what this is and what it's used for? (I'm not sure I'm following you...).

Reply:
this `value` is redundant, so it's dropped

Reply:
Great! :)